agent/stats/engine_integ_test.go

```diff
@@ -659,3 +659,79 @@ func testNetworkModeStats(t *testing.T, networkMode string, statsEmpty bool) {
 	validateIdleContainerMetrics(t, engine)
 	validateEmptyTaskHealthMetrics(t, engine)
 }
+
+func TestBlockStats(t *testing.T) {
```

Comment:
how about renaming it to StorageStats and in the event stream below?

Reply:
I'll do that.